GPCS4/Loader/ELFMapper.cpp

```diff
@@ -261,7 +261,7 @@ bool ELFMapper::mapImageIntoMemroy()
 			if (phdr.p_flags & PF_X)
 			{
 				retVal = mapCodeSegment(phdr);
-				LOG_DEBUG("code segment at 0x%08x size=%ld", info.pCodeAddr,
+				LOG_DEBUG("code segment at 0x%x size=%ld", info.pCodeAddr,
```

Comment:
Maybe we should not use %x here because pointers and unsigned ints are not the same size in x64. What about %p?

Reply:
%p shows a different way on differ platforms. For example, on macOS it shows as "0x....", on Windows without "0x"